optimizationmodule/Function.go

```diff
@@ -0,0 +1,53 @@
+package optimizationmodule
+
+import (
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"slices"
+)
+
+type Function interface {
+	Call(wrapper *openrtb_ext.RequestWrapper) (string, error)
+}
+
+// will be used in Build rules trie
+func NewFunction(name string, params []string) Function {
+	switch name {
+	case "deviceCountry":
+		return NewDeviceCountry(params)
+	case "setDeviceIP":
+		return NewSetDevIp(params)
+	default:
+		return nil
+	}
+}
+
+type DeviceCountry struct {
+	CountryCodes []string
+}
+
+func (dc *DeviceCountry) Call(wrapper *openrtb_ext.RequestWrapper) (string, error) {
+	contains := slices.Contains(dc.CountryCodes, wrapper.Device.Geo.Country)
+	//convert result to string
+	if contains {
+		return "yes", nil
+	}
+	return "no", nil
+}
```

Comment:
Looks good. We can make the yes / no constants to reuse in other functions... or should these be "true" and "false"?